ddtrace/contrib/tornado/__init__.py

```diff
@@ -0,0 +1,73 @@
+"""
+The Tornado integration traces all ``RequestHandler`` defined in a Tornado web application.
+Auto instrumentation is available using the ``trace_app`` function as follows::
+
+    import tornado.web
+    import tornado.gen
+    import tornado.ioloop
+
+    from ddtrace import tracer
+    from ddtrace.contrib.tornado import trace_app
+
+    # create your handlers
+    class MainHandler(tornado.web.RequestHandler):
+        @tornado.gen.coroutine
+        def get(self):
+            self.write("Hello, world")
+
+    # create your application
+    app = tornado.web.Application([
+        (r'/', MainHandler),
+    ])
+
+    # trace your application before the execution
+    trace_app(app, tracer, service='tornado-site')
+
+    # and run it as usual
+    app.listen(8888)
+    tornado.ioloop.IOLoop.current().start()
+
+When a ``RequestHandler`` is hit, a request span is automatically created and attached
+to the current ``request`` object, so that it can be used in the application code::
+
+    class MainHandler(tornado.web.RequestHandler):
+        @tornado.gen.coroutine
+        def get(self):
+            ctx = getattr(self.request, 'datadog_context')
```

Comment:
About the API, the `tracer.get_call_context()` can be used and it will work. If it doesn't, it means that `ctx = getattr(self.request, 'datadog_context')` is wrong too. I attached the `context` just to use the request as a carrier, but yea... we're using a `StackContext` to transfer the right context to the right logical execution so this could be seen as redundant.

About docs, if we say: "**to retrieve your context you must use our `tracer.get_call_context()`` API**", then we can remove any occurrence here. It could also applies for `aiohttp`.